.github/ISSUE_TEMPLATE/new_bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
+about: Identification of a new issue
+title: 'New Bug'
+labels: 
+assignees: '@abbylindsay'
+
+---
+
+**Describe the bug- why is it wrong?:**
+<!-- A clear and concise description of what you/the participant is seeing and why it is a problem. -->
+
+**How was this bug identified:**
+<!-- Who saw it? What was the context? -->
+
+**How to reproduce?:**
+<!-- If no instructions, you MUST provide an explaination as to why not. -->
+
+**Troubleshooting completed so far:**
+<!-- What has been done thus far to resolve/investigate the bug? -->
+
+**Expected behavior?:**
+<!-- Be descriptive: what should be happening? Provide examples if possible. -->
+
+**Screenshot(s):**
+<!-- Show the bug. -->
+
+**Priority level?:**
+<!-- 1- a V&V will not be run for this issue alone. 2- Want to have 3- Causing minor impact to study/participant 4- Causing big impact to study/participant 5- BAD BAD BAD -->
+
+**Additional Context:**
+<!-- The more info the better -->
```

Comment:
Do you want to capture a "Component" dropdown here? E.g., "Sensor", "Reports", "Portal", "Tablet"?

Reply:
do you think we need that?  i think that would be clear by the issue. i dont want ops to have an exhaustive report each time they fill out an issue request (yes a drop down is easy but i think its the principle here of deciding what is actually a value-add question)